pkgs/top-level/perl-packages.nix

```diff
@@ -1898,6 +1898,19 @@ let self = _self // overrides; _self = with self; {
     };
   };
 
+  Clipboard = buildPerlPackage {
+    name = "Clipboard-0.13";
+    src = fetchurl {
+      url = mirror://cpan/authors/id/K/KI/KING/Clipboard-0.13.tar.gz;
+      sha256 = "eebf1c9cb2484be850abdae017147967cf47f8ccd99293771517674b0046ec8a";
+    };
+    meta = {
+      description = "Cliboard - Copy and Paste with any OS";
+      license = "perl";
```

Comment:
The author is [somewhat nebulous](https://metacpan.org/pod/Clipboard#COPYRIGHT) about the license. He links the Artistic License 1 but also says that it is licensed like Perl itself, which in turn is dual licensed with the Artistic License 1 and the GPL, so I'm not really clear on what licenses to use.